ci/gpu/build.sh

```diff
@@ -61,59 +61,89 @@ conda config --show-sources
 conda list --show-channel-urls
 
 ################################################################################
-# BUILD - Build ucx-py
+# TEST - Run py.tests for ucx-py
 ################################################################################
+function run_tests() {
+    if hasArg --skip-tests; then
+        gpuci_logger "Skipping Tests"
+    else
+        gpuci_logger "Check GPU usage"
+        nvidia-smi
+
+        gpuci_logger "Check NICs"
+        awk 'END{print $1}' /etc/hosts
+        cat /etc/hosts
+
+        gpuci_logger "UCX Version and Build Configuration"
+        ucx_info -v
+
+        gpuci_logger "Python py.test for ucx-py"
+        cd $WORKSPACE
+
+        # list test directory
+        ls tests/
+
+        # Setting UCX options
+        export UCXPY_IFNAME=eth0
+        export UCX_MEMTYPE_CACHE=n
+        export UCX_TLS=tcp,cuda_copy,sockcm
+        export UCX_SOCKADDR_TLS_PRIORITY=sockcm
+
+        # Test with TCP/Sockets
+        gpuci_logger "TEST WITH TCP ONLY"
+        py.test --cache-clear -vs --ignore-glob tests/test_send_recv_two_workers.py tests/
+        py.test --cache-clear -vs ucp/_libs/tests
+
+        # Test downstream packages, which requires Python v3.7
+        if [ $(python -c "import sys; print(sys.version_info[1])") -ge "7" ]; then
+            gpuci_logger "TEST OF DASK/UCX"
+            py.test --cache-clear -vs `python -c "import distributed.protocol.tests.test_cupy as m;print(m.__file__)"`
+            py.test --cache-clear -vs `python -c "import distributed.protocol.tests.test_numba as m;print(m.__file__)"`
+            py.test --cache-clear -vs `python -c "import distributed.protocol.tests.test_rmm as m;print(m.__file__)"`
+            py.test --cache-clear -vs `python -c "import distributed.protocol.tests.test_collection_cuda as m;print(m.__file__)"`
+            py.test --cache-clear -vs `python -c "import distributed.comm.tests.test_ucx as m;print(m.__file__)"`
+            py.test --cache-clear -vs `python -c "import distributed.tests.test_nanny as m;print(m.__file__)"`
+            py.test --cache-clear -m "slow" -vs `python -c "import distributed.comm.tests.test_ucx as m;print(m.__file__)"`
```

Comment:
Related to https://github.com/dask/distributed/issues/4902#issuecomment-858996585, it might be worth adding 
```
--rootdir=`python -c "import distributed as m;print(m.__path__[0])"`
```

here to see if that helps resolve the issue